shared/app.js

```diff
@@ -70,6 +72,8 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
+      var ClientRouter = options.ClientRouter || require('app/router');
```

Comment:
i think this needs the attributes.ClientRouter that way it will get the existing router as well.